model/Human-GEM.yml

```diff
@@ -5,8 +5,8 @@
     full_name: "Generic genome-scale metabolic model of Homo sapiens"
     version: ""
     date: "2023-04-24"
-    authors: "Jonathan Robinson, Hao Wang, Pierre-Etienne Cholley, Pinar Kocabas"
-    email: "jonrob@chalmers.se"
+    authors: ""
+    email: ""
```

Comment:
Interesting idea @haowang-bioinfo. Instead of having the field empty, how about removing the fields completely? The same goes for the `organization` field below.

Reply:
> how about removing the fields completely? The same goes for the organization field below.

make sense to me

Reply:
removed the fields in 14695c94d709a74c076d5c1cb92591bd750f1283